test/blocks/merch-card/merch-card.test.js

```diff
@@ -497,3 +497,18 @@ describe('Viewport Responsiveness without Sinon', () => {
     expect(bigPrice.style.fontSize).to.equal('16px');
   });
 });
+
+describe('Product Merch Card', () => {
+  it('Supports Product Merch card with callout', async () => {
+    document.body.innerHTML = await readMockText('/test/blocks/merch-card/mocks/product.html');
+    const merchCard = await init(document.querySelector('.product'));
+
+    // Assert
```

Comment:
I guess we can remove this comment, and the one below, it's pretty clear from the code what it does :)